Cargo.toml

```
@@ -9,72 +9,63 @@ license = "MIT"
 description = "Parsing engine for Corn, a simple and pain-free configuration language."
 repository = "https://github.com/JakeStanger/corn"
 categories = ["config", "parsing"]
-keywords = ["configuration", "language", "wasm", "pest", "peg"]
-authors = ["Jake Stanger <mail@jakestanger.com>"]
+keywords = ["configuration", "language", "wasm", "peg"]
+authors = [
+    "Jake Stanger <mail@jakestanger.com>",
```

This seems to be an issue in the repo anyway and I've no idea how it happened because I've never owned the domain but this is wrong:

```suggestion
    "Jake Stanger <mail@jstanger.dev>"
```